pom.xml

```diff
@@ -129,6 +129,36 @@
   </pluginRepositories>
 
   <build>
+    <pluginManagement>
```

Comment:
unrelated to the change, but due to eclipse-m2e/m2e-core/issues/1291 would cause an error in the project in eclipse.